simulators/history/rpc-compat/src/main.rs

```diff
@@ -469,10 +600,16 @@ dyn_async! {
 
 dyn_async! {
     // test that a node will return a AbsentContent via RecursiveFindContent when the data doesn't exist
-    async fn test_recursive_find_content_content_absent<'a> (client: Client) {
+    async fn test_recursive_find_content_content_absent<'a>(clients: Vec<Client>, _: Option<Vec<(String, String)>>) {
```

Comment:
I don't think we should add history to every single function name. ``simulators/history/rpc-compat/src/main.rs`` because in the ``simulators/history`` folder we will only ever test the history network only.